server/database/schema.sql

```diff
@@ -1,12 +1,45 @@
 create table user (
   id int unsigned primary key auto_increment not null,
-  email varchar(255) not null unique,
-  password varchar(255) not null
+  username varchar(80) not null,
+  firstname varchar(80),
+  lastname varchar(80),
+  city varchar(80) not null,
+  zipcode int not null,
+  email varchar(255) not null,
+  password varchar(255) not null,
+  role varchar(80) not null,
+  registration_date date not null
 );
 
-create table item (
+create table art (
   id int unsigned primary key auto_increment not null,
-  title varchar(255) not null,
+  title varchar(80),
+  information varchar(255),
+  latitude DECIMAL (10,8) not null,
```

Comment:
Pourquoi le champ latitude et longitude sont différentes ?

Reply:
J'ai modifié latitude et longitude, voici la doc sur laquelle je me base :
[https://stackoverflow.com/questions/1196415/what-datatype-to-use-when-storing-latitude-and-longitude-data-in-sql-databases](url)